.github/workflows/tests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.10", 3.11]
```

Comment:
Does "3.11" need quotes?

Reply:
no, 3.10 has quotes just so it doesn't show up as 3.1 in the github actions sidebar